### ISSUE TITLE
On data preprocessing

### ISSUE THREAD

**Issue by MeatballCat** [Megvii-BaseDetection/YOLOX#691]
Hello, I'd like to ask about Max in trantransform class_ What is the role of the labels parameter? Why is my image data transferred into the neural network without normalization parameters, but simply converted into tensor. Thank you.

**Comment by MeatballCat** [comment]
sorry, I have some problems with the above writing. what's role of max_labels in TrainTransform class?

**Comment by awarebayes** [comment]
Yeah, same question, would be nice if someone elaborated...

**Comment by awarebayes** [comment]
Ah, they are simply using batch norm before input, and do not use imagenet pretrained weights

**Comment by FateScript** [comment]
`max_labels` is decided by dataset you are training. Since different images have different number of boxes, all box annotations should be padded into the same size decided by `max_label`.
